<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="doctorNameEditText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="doctorNameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>Podaj imię:</string>
   </property>
  </widget>
  <widget class="QLabel" name="doctorSurnameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>Podaj nazwisko:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctorSurnameEditText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="doctorSpecjalizationLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>Podaj specjalizację:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctorSpecjalizationEditText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="registerDoctorLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(34, 148, 255)</string>
   </property>
   <property name="text">
    <string>DODAWANIE LEKARZA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 148, 255); color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
